### DISCUSSION TITLE
为什么post总是返回根目录

### DISCUSSION THREAD

**Discussion by Yue-cn** [AlistGo/alist#473] [Q&A]
https://mcmirror.zclubserver.top/api/public/path

post内容:
{"path":"/Teambition"}

**Comment by xhofe** [comment]
哪里总是返回根目录了
![image](https://user-images.githubusercontent.com/36558727/152301204-aede9f2c-47f6-4085-aaea-5edca49905ae.png)

**Reply by Yue-cn** [reply]
抱歉,目录搞错了是
/本地下载

**Reply by xhofe** [reply]
![image](https://user-images.githubusercontent.com/36558727/152301580-cbe7b5a8-7f9e-4d7a-a62a-fdcb5a8163b6.png)

**Comment by Yue-cn** [comment]
![~{MS`WGWVCW$)LN3352Q_18](https://user-images.githubusercontent.com/79188080/152301602-2697a1ab-a88a-4373-a20c-f5393177a1df.png)

**Reply by xhofe** [reply]
![image](https://user-images.githubusercontent.com/36558727/152301852-eee06ddb-e347-4520-b677-92ddea92c3cf.png)

**Reply by Yue-cn** [reply]
done....

**Reply by z0z0r4** [reply]
> done....

Hhhhhh
